spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-bus-config-sample/pom.xml

```diff
@@ -14,7 +14,7 @@
     <name>Spring Framework on Google Cloud Code Sample - Pub/Sub Bus Configuration Management</name>
 
     <properties>
-        <spring-cloud-config.version>4.1.3</spring-cloud-config.version>
+        <spring-cloud-config.version>4.2.0</spring-cloud-config.version>
```

Comment:
This PR should also contain some updates to renovate.json (or similar files) specifying the new version ranges that are supported for each of the branches.

Reply:
I decided to separate the branch setup in https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/3436/files